build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   ws,
   specs2 % Test,
   "com.gu" %% "pan-domain-auth-play_2-4-0" % "0.2.7",
-  "com.amazonaws" % "aws-java-sdk" % "1.7.5"
+  "com.amazonaws" % "aws-java-sdk" % "1.10.50"
```

Comment:
I think adding the whole sdk is fine if you're the last link in the chain. We had problems in `facia-tool` and `frontend` because some dependencies were pulling in incompatible versions of e.g. `kinesis` even if they didn't need it.

So it's good to know that if you're writing a library you should declare only what you really need, otherwise it doesn't matter as much.